src/app/components/NavBarBg.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { Box } from "@chakra-ui/react";
+import { NavBar } from "./NavBar";
+
+export const NavBarBg = () => {
```

Comment:
Maybe even "TransparentNavBarBg" so that you'll know when you come back to it in 3 months.
I would also comment with words to remind yourself in the future. something like // no rgba() chakra support, so a hacky solution for now